docs/plans/2026-07-19-photo-filter-prototype/prototype.css

```
@@ -0,0 +1,603 @@
+  --text-dim: #5a8890;
+  --text-faint: #3d6670;
+  --accent: #24e5ca;
+  --accent-hover: #5af0da;
+  --accent-dim: #0a4a42;
+  --accent-text: #0a1f2e;
+  --warning: #f0c040;
+  --danger: #e76a5f;
+  --info: #74b8d4;
+  --shadow: 0 18px 50px rgba(0, 0, 0, .34);
+  --thumb-size: 200px;
+  color-scheme: dark;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)